api/gateway/stripe/tests/StripeMarketplaceTest.cfc

```diff
@@ -0,0 +1,782 @@
+	</cffunction>
+	
+	<cffunction name="testDeleteDefaultForCurrencyBankAccountFails" access="public" returntype="void" output="false" mxunit:dataprovider="gateways">
+		<cfargument name="gw" type="any" required="true" />
+
+			<!--- Create Connected Account --->
+		<cfset offlineInjector(gw, this, "mock_create_account_ok", "doHttpCall") />
+		<cfset connectedAccount = gw.marketplaceCreateConnectedAccount() />
+		<cfset standardResponseTests(response = connectedAccount, expectedObjectName = "account", expectedIdPrefix="acct_") />
+
+			<!--- Create Bank Account --->
+		<cfset offlineInjector(gw, this, "mock_create_bank_accounts_ok", "doHttpCall") />
+		<cfset bankAccount = gw.marketplaceCreateBankAccount(connectedAccount = connectedAccount.getParsedResult().id, account = createAccount()) />
+		<cfset standardResponseTests(response = bankAccount, expectedObjectName = "bank_account", expectedIdPrefix="ba_") />
+
+			<!--- Delete Bank Account --->
+		<cfset offlineInjector(gw, this, "mock_delete_bank_accounts_fail", "doHttpCall") />
+		<cfset delete = gw.marketplaceDeleteBankAccount(connectedAccount = connectedAccount.getParsedResult().id, bankAccountId = bankAccount.getParsedResult().id) />
+		<cfset standardErrorResponseTests(response = delete, expectedStatusCode="400", expectedErrorType = "invalid_request_error") />
+	</cffunction>
+
+		<!--- Identity Verification Tests --->
+	<cffunction name="testUploadIdentityFile" access="public" returntype="void" output="false" mxunit:dataprovider="gateways">
+		<cfargument name="gw" type="any" required="true" />
+		<cfset offlineInjector(gw, this, "mock_upload_identity_file_ok", "doHttpCall") />
+		<cfset uploadFile = gw.marketplaceUploadIdentityFile(file = "C:\inetpub\pukka\msr\shared\cfpayment\api\gateway\stripe\tests\sample_driving_license_usa.jpg") />
+		<cfset standardResponseTests(response = uploadFile, expectedObjectName = "file_upload", expectedIdPrefix="file_") />
+	</cffunction>
+
+	<cffunction name="testAttachFileToAccount" access="public" returntype="void" output="false" mxunit:dataprovider="gateways">
+		<cfargument name="gw" type="any" required="true" />
+
+			<!--- Create Connected Account --->
+		<cfset offlineInjector(gw, this, "mock_create_account_ok", "doHttpCall") />
+		<cfset connectedAccount = gw.marketplaceCreateConnectedAccount() />
+		<cfset standardResponseTests(response = connectedAccount, expectedObjectName = "account", expectedIdPrefix="acct_") />
+
+			<!--- Upload Identity File --->
+		<cfset offlineInjector(gw, this, "mock_upload_identity_file_ok", "doHttpCall") />
+		<cfset argumentCollection = structNew()>
+		<cfset argumentCollection.file = "C:\inetpub\pukka\msr\shared\cfpayment\api\gateway\stripe\tests\sample_driving_license_usa.jpg" />
+		<cfset argumentCollection.accountSecret = connectedAccount.getParsedResult().keys.secret />
```

Comment:
this test won't run in another environment - use expandPath() to grab the current working directory and make this test portable